tox.ini

```diff
@@ -106,6 +106,7 @@ commands =
     bash {toxinidir}/.travis/runcoveralls.sh {posargs}
 
 [molecule_common]
+passenv = MOLECULE_* LSR_MOLECULE_*
```

Comment:
Why is this needed? Looks to me like `passenv = *` from `[testenv]` should already cover this.

Reply:
> Why is this needed? Looks to me like `passenv = *` from `[testenv]` should already cover this.

Looks like it isn't needed - I guess I was confused by all of the `passenv` customization in the other tasks/envs.
So then the question becomes - do we want to restrict the environment for running molecule, in the same manner that we restrict the environment for the other tasks/envs, or do we leave it wide open?

Reply:
IMHO restricting the environment prevents from unintended environment poisoning. OTOH if some environment variables are forgotten to be passed it can have a negative impact on tests.